templates/cloudformation/apps/aem/full-set/monitoring.yaml

```diff
@@ -15,37 +15,429 @@ Resources:
+                  [ "...", "${orchestratorAutoScalingGroupName}", "Filesystem", "/dev/xvdb" ],
+                  [ "...", "${choasMonkeyAutoScalingGroupName}", "Filesystem", "/dev/xvdb" ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "ap-southeast-2",
+                "title": "DiskSpaceUtilization on Data Volume /mnt/ebs1"
+              }
+            },
+            {
+              "type": "metric",
+              "x": 12,
+              "y": 33,
+              "width": 12,
+              "height": 6,
+              "properties": {
+                "metrics": [
+                  [ "System/Linux", "DiskSpaceUtilization", "MountPath", "/", "InstanceId", "${authorPrimaryInstance}", "Filesystem", "/dev/xvda2" ],
+                  [ "...", "${authorStandbyInstance}", "Filesystem", "/dev/xvda2" ],
+                  [ "System/Linux", "DiskSpaceUtilization", "MountPath", "/", "AutoScalingGroupName", "${authorDispatcherAutoScalingGroupName}", "Filesystem", "/dev/xvda2" ],
+                  [ "...", "${publishAutoScalingGroupName}", "Filesystem", "/dev/xvda2" ],
+                  [ "...", "${publishDispatcherAutoScalingGroupName}", "Filesystem", "/dev/xvda2" ],
+                  [ "...", "${orchestratorAutoScalingGroupName}", "Filesystem", "/dev/xvda2" ],
+                  [ "...", "${choasMonkeyAutoScalingGroupName}", "Filesystem", "/dev/xvda2" ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "ap-southeast-2",
+                "title": "DiskSpaceUtilization on root volume"
+              }
+            },
+            {
+              "type": "metric",
+              "x": 0,
+              "y": 39,
+              "width": 12,
+              "height": 6,
+              "properties": {
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "ap-southeast-2",
+                "title": "SQS AEM ASG Event queue - Available Messages",
+                "start": "-PT3H",
+                "end": "P0D",
+                "metrics": [
+                  [ "AWS/SQS", "ApproximateNumberOfMessagesVisible", "QueueName", "${aemAsgEventQueueName}" ]
+                ]
+              }
+            },
+            {
+              "type": "metric",
+              "x": 12,
+              "y": 39,
+              "width": 12,
+              "height": 6,
+              "properties": {
+                "metrics": [
+                  [ "collectd", "GenericJMX.delay.seconds_since_last_success", "FixedDimension", "${stackPrefix}-author-standby", "Host", "${authorStandbyInstance}", "PluginInstance", "standby-status", { "period": 60, "stat": "Maximum" } ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "ap-southeast-2",
```

Comment:
This region should be coming from parameter just like the example.

Reply:
Thanks for highlighting. Replaced region value with variable `${awsRegion}`